README.md

```diff
@@ -1,5 +1,33 @@
-# Hello Microverse Project
-My first project at Microverse
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
```

Comment:
- [ **OPTIONAL** ] For clean code and as mentioned in this template, kindly ensure that you remove all the comments and instructions after you have used the template